addons/cat-blocks/addon.json

```
@@ -1,6 +1,6 @@
 {
   "name": "Cat blocks",
-  "description": "Brings back editor cat hat blocks from April Fools 2020.",
+  "description": "Brings back editor cat hat blocks from April Fools 2020-2023.",
```

```suggestion
  "description": "Brings back editor cat hat blocks that were first introduced for April Fools' Day 2020.",
```
This mitigates the issue I mentioned because it refers only to 2020 but still acknowledges that future versions do exist. While at it I, I also improved the name of April Fools' Day, but you don't have to accept that it you prefer the previous name.